### PR TITLE
Update Istio documentation according to changes in Helm Chart v3

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
## Purpose of this pull request

Update Istio documentation according to changes in Helm Chart v3

Issue number:  https://jira.kumoroku.com/jira/browse/SUMO-206934

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/saas-cloud/istio.md [hunk 1]
```diff
@@ -36,26 +34,34 @@ namespace=istio-system cluster={{cluster}}
 | transpose row _timeslice column response_code
 ```
 
-
 ## Collecting Logs and Metrics for the Istio App
 
-This section provides instructions for collecting logs and metrics for the Sumo App for Istio. Logs and metrics are collected with the Sumo Logic Helm chart. Istio [sample metrics](#Sample-Metrics) and [sample log messages](#Sample-Log-Messages) are also provided, along with a [query sample](#Query-Sample).
+This section provides instructions for collecting logs and metrics for the Sumo App for Istio. Logs and metrics are collected with the [Sumo Logic Kubernetes Collection Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection).
+If you do not have Kubernetes Collection set up, go [here](/docs/observability/kubernetes) to learn more and check [Kubernetes Quickstart](/docs/observability/kubernetes/quickstart.md) to see installation instruction.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/istio.md:40]:
```suggestion
If you've not yet set up Kubernetes Collection, visit our [Kubernetes](/docs/observability/kubernetes) and [Kubernetes Quickstart](/docs/observability/kubernetes/quickstart) docs to learn how to install.
```

File: docs/integrations/saas-cloud/istio.md [hunk 2]
```diff
@@ -36,26 +34,34 @@ namespace=istio-system cluster={{cluster}}
...
+  The [Sumo Logic Kubernetes Collection Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection) will automatically capture the logs from stdout and will send the logs to Sumologic.
 
 Metrics Collection:
-1. If you did install using the Sumo Logic Helm chart:
-   * Update the helm chart values file with the following config:
+
+1. If you did install using the [Sumo Logic Kubernetes Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection):
+
+   * Update the Helm Chart values file with the following config:
+
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/istio.md:55]:
```suggestion
1. If you installed using the [Sumo Logic Kubernetes Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection):
   * Update the Helm Chart values file with the following config:
 
```

File: docs/integrations/saas-cloud/istio.md (hunk 1 above)
Comment (kkujawa-sumo) [reply docs/integrations/saas-cloud/istio.md]: changed

File: docs/integrations/saas-cloud/istio.md (hunk 2 above)
Comment (kkujawa-sumo) [reply docs/integrations/saas-cloud/istio.md]: changed

**Pull request merged**